presto-docs/src/main/sphinx/release/release-0.220.rst

```diff
@@ -0,0 +1,55 @@
+=============
+Release 0.219
+=============
+
+General Changes
+---------------
+* Fix an issue when executing spatial join on top of a right join. See :issue:`12672`.
+* Fix an issue where ``TRY`` cannot catch the failures when ``ROW`` and ``ARRAY`` with ``NULL`` elements fail to compare.
+* Lazily build the hash table for map.
+
+Raptor Changes
+--------------
+* Integrate the optimized ORC writer into Raptor. Add configuration property ``storage.orc.optimized-writer-stage`` to enable and configure the optimized writer.
+  To switch to the optimized ORC writer is only backward compatible but not forward compatible.
+  Turning on the optimized ORC writer should work on an existing Raptor server. However, switching it back to Hadoop ORC writer can fail Raptor background job.
+* Integrate ``ZSTD`` compression for file writer. Add configuration property ``storage.orc.compression-kind`` to switch between ``SNAPPY`` and ``ZSTD``.
+  This is only applied when optimized ORC writer is turned on.
+
+
+Hive Connector Changes
+----------------------
+* Add ``compression_codec`` session property to set the compression codec when writing files. Possible values are ``NONE``, ``SNAPPY``, and ``GZIP``.
+* Add experimental support for partial merge pushdown, which optimizes plan for queries joining over two tables with mismatched but compatible bucket counts.
+  This feature can be enabled by session property ``experimental.optimizer.partial-merge-pushdown-strategy``, and is aiming to supersede ``hive.optimize-mismatched-bucket-count``.
+  See :issue:`12611`.
+* Improve writing to bucketed table by cache Hive Metastore.
```

Comment:
what's improved? is this a user-facing change?

Reply:
@picrinite Could you add more details about this?

Reply:
Changed to
```
Improve query performance by caching partition information to
avoid duplicate Hive Metastore calls when writing bucketed tables.
```

Reply:
I think this would be too complicated for most people to parse.  How about

```
Improve performance of writes to bucketed Hive tables.
```

Reply:
What about
```
Improve query performance when writing bucketed tables.
```

@rschlussel How do we balance succinctness vs. context/details?

Reply:
I like @tdcmeehan's suggestion or your second suggestion. Our release notes focus on the user, so the implementation details don't necessarily matter, but we should always include context about what kinds of queries are affected and how.  
Here's what context that I feel is important to provide for some common scenarios:
* bug fixes -- what kinds of queries encountered the issue, what would the user have seen before (failures with a particular error message, wrong results, slow performance), and if it's not obvious, then say what the behavior is now.
* new features -- basic description of the feature and how to turn it on and link to docs about it
* performance improvement- what kinds of queries will see improved performance and possibly what will be improved about it (e.g. is it a walltime improvement or memory improvement, etc.).